wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -140,11 +145,17 @@ extension ZMConversation {
                     completion(.failure(ConversationFetchError.unknown))
                     return
                 }
-                let fetchResult = (conversationId, conversationName)
+
+                var passwordProtected: Bool?
+                if apiVersion.rawValue >= 4 && apiVersion.rawValue <= 6 {
+                    passwordProtected = payload["has_password"] as? Bool
+                }
```

Comment:
Would this break for upcoming api versions 7 and later?
A second point is that `APIVersion` is `Comparable`, which can be used without `rawValue`:
`apiVersion >= .v4`

Reply:
I guess what I can do is only have `apiVersion >= .v4` and remove the `apiVersion.rawValue <= 6`

Reply:
Done! https://github.com/wireapp/wire-ios/commit/f5b99ad83f67dd76eb9f09cc9456ad33f9d7bcc8